server/apps/scaling-pulsar-smtp/src/main/java/org/apache/james/Main.java

```diff
@@ -86,14 +91,24 @@ public class Main implements JamesServerMain {
     );
 
     public static final Module SERVER_CORE_MODULES = Modules.combine(
-            new MailetProcessingModule(),
             new DefaultProcessorsConfigurationProviderModule(),
-            new CassandraSessionModule(),
-            new CassandraDomainListModule(),
-            new CassandraRecipientRewriteTableModule(),
-            new CassandraUsersRepositoryModule(),
-            new CassandraMailRepositoryModule(),
-            new CassandraMetricsModule(),
+            new CassandraSessionModule(),// to remove
+            //        new JPAEntityManagerModule(),
```

Comment:
Don't forget those two as well. Remove? Keep?

Reply:
@chibenwa  asked the [same question](https://github.com/apache/james-project/pull/1257#discussion_r997248212) I'll answer there and resolve this comment to keep things simple